client/modules/IDE/components/Header/Nav.unit.test.jsx

```diff
@@ -1,62 +1,11 @@
 import React from 'react';
-import { render, reduxRender } from '../../../../test-utils';
+import { reduxRender } from '../../../../test-utils';
 
-import Nav, { NavComponent } from './Nav';
+import Nav from './Nav';
 
 // jest.mock('../i18n');
 
 describe('Nav', () => {
-  const props = {
-    newProject: jest.fn(),
-    saveProject: jest.fn(),
-    autosaveProject: jest.fn(),
-    exportProjectAsZip: jest.fn(),
-    cloneProject: jest.fn(),
-    user: {
-      authenticated: true,
-      username: 'new-user',
-      id: 'new-user'
-    },
-    project: {
-      id: 'new-project',
-      owner: {
-        id: 'new-user'
-      }
-    },
-    logoutUser: jest.fn(),
-    newFile: jest.fn(),
-    newFolder: jest.fn(),
-    showShareModal: jest.fn(),
-    showErrorModal: jest.fn(),
-    unsavedChanges: false,
-    warnIfUnsavedChanges: jest.fn(),
-    showKeyboardShortcutModal: jest.fn(),
-    cmController: {
-      tidyCode: jest.fn(),
-      showFind: jest.fn(),
-      findNext: jest.fn(),
-      findPrev: jest.fn(),
-      showReplace: jest.fn()
-    },
-    startSketch: jest.fn(),
-    stopSketch: jest.fn(),
-    setAllAccessibleOutput: jest.fn(),
-    showToast: jest.fn(),
-    setToastText: jest.fn(),
-    rootFile: {
-      id: 'root-file'
-    },
-    t: jest.fn(),
-    setLanguage: jest.fn(),
-    language: 'en-US',
-    isUserOwner: true
-  };
-
-  it('renders correctly', () => {
-    const { asFragment } = render(<NavComponent {...props} />);
-    expect(asFragment()).toMatchSnapshot();
-  });
-
```

Comment:
It is sooo much cleaner with all those props gone! 🧹

Reply:
yes it is 😂